### DISCUSSION TITLE
I2U¥ Instrumental Group

### DISCUSSION THREAD

**Discussion by lancealotlink** [Anjok07/ultimatevocalremovergui#456] [Show and tell]
<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/qRbfGy0F/image.jpg' border='0' alt='image'/></a>

I2U¥ Has done many tracks and is in the process of a 6000 Track rip project that you could Demucs and split for use as the vocals are already removed Latest Example rips below but I2U¥ has done many 100s before the project currently.

Everything on youtube is shared as mp3 freely in the I2U¥ Instrumentals channel on Telegram

Youtube channel
https://www.youtube.com/@i2u26/featured

Telgram channel
https://t.me/+4mR1owB87b9kMmVl

I2U¥ Telgram Chat Group
https://t.me/+9_gm0Pj8ias4ZWVl

Latest Example rips

<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/Df6QB690/photo-2023-03-19-00-25-29.jpg' border='0' alt='photo-2023-03-19-00-25-29'/></a>

<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/9F34P7M0/men.jpg' border='0' alt='men'/></a>

<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/K80vWkZn/classic-rock.jpg' border='0' alt='classic-rock'/></a>
<a href='https://postimg.cc/qzWfTdDd' target='_blank'><img src='https://i.postimg.cc/2SSCnk43/Untitled.jpg' border='0' alt='Untitled'/></a>

**Comment by thecatontheceiling** [comment]
please don't self advertise in discussions.
